opsmngr/unauth_users.go

```diff
@@ -68,25 +68,6 @@ type WhitelistOpts struct {
 	Whitelist []string `url:"whitelist"`
 }
 
-// User wrapper for a user response, augmented with a few extra fields
-type User struct {
-	Username     string        `json:"username"`
-	Password     string        `json:"password,omitempty"`
-	FirstName    string        `json:"firstName,omitempty"`
-	LastName     string        `json:"lastName,omitempty"`
-	EmailAddress string        `json:"emailAddress,omitempty"`
-	ID           string        `json:"id,omitempty"`
-	Links        []*atlas.Link `json:"links,omitempty"`
-	Roles        []*UserRole   `json:"roles,omitempty"`
-}
-
-// UserRole denotes a single user role
-type UserRole struct {
-	RoleName string `json:"roleName"`
-	GroupID  string `json:"groupId,omitempty"`
-	OrgID    string `json:"orgId,omitempty"`
-}
-
```

Comment:
[q] I see you're moving this, but was the struct not being used anywhere before? What happens when creating the first user now, does it use the User struct from the Users service?

Reply:
Yes! This is just the type definitions, which are shared amongst these files. Felt more appropriate to define the Users in the user endpoint, and then this unauth endpoint can borrow from it